### PR TITLE
getting started providers: route to various places

### PR DESCRIPTION
**Pull request by gwarf**
Some simple text to remove a placeholder.

Wondering if notebooks doc shouldn't go under internal services, is it really something to be deployed by service providers themselves?

### CODE REVIEW

**Comment by enolfc** [comment]
The notebooks docs is about the deployment of the instances, it's true that there is 1/2 providers, but where would you move it to?

File: content/en/providers/getting-started/_index.md [hunk 1]
```diff
@@ -5,6 +5,17 @@ type: docs
 weight: 10
 ---
 
-{{% pageinfo %}}
-This is a placeholder page.
-{{% /pageinfo %}}
+The pages under this section depict how to join the EGI infrastructure as
+a service providers offering innovative services to the European Research Area.
+
+> Interested in integrating with Check-in? Head to the
```
Comment (enolfc) [inline content/en/providers/getting-started/_index.md:11]: what about IdPs?

Comment (gwarf) [reply content/en/providers/getting-started/_index.md:11]: the integrating was expected to cover both IdP and SP in fact. Maybe we could be more clear and point to the sub sections for both aspects.

Comment (gwarf) [reply content/en/providers/getting-started/_index.md]: I've updated to cover this.

**Comment by gwarf** [comment]
> The notebooks docs is about the deployment of the instances, it's true that there is 1/2 providers, but where would you move it to?

Let's keep it there for now.

**Review by enolfc — APPROVED** [review]
👍

**Pull request merged**